op-deployer/pkg/deployer/bootstrap/implementations.go

```diff
@@ -70,10 +70,8 @@ func (c *ImplementationsConfig) Check() error {
 	if c.ArtifactsLocator == nil {
 		return errors.New("artifacts locator must be specified")
 	}
-	if c.ArtifactsLocator.IsTag() {
-		c.L1ContractsRelease = c.ArtifactsLocator.Tag
-	} else {
-		c.L1ContractsRelease = "dev"
+	if c.L1ContractsRelease == "" {
+		return errors.New("l1 contracts release must be specified")
```

Comment:
I think we should also ensure that the release matches the tag.

Reply:
Why don't we do the following:
```
    if c.ArtifactsLocator.IsTag() {
        c.L1ContractsRelease = c.ArtifactsLocator.Tag
	} else if c.L1ContractsRelease == "" {
        return errors.New("l1 contracts release must be specified if not using an artifacts tag")
	}
```

Reply:
This LGTM

Reply:
ok updated